css-conditional-5/scroll_state_explainer.md

```diff
@@ -147,13 +147,36 @@ a carousel.
 
 - [Author request on github](https://github.com/w3c/csswg-drafts/issues/7430)
 
-### Overflowing
+### Scrollable
 
-Query whether a container has scrollable overflow. Can be used to indicate
+Query whether a container has [scrollable overflow](https://drafts.csswg.org/css-overflow-3/#scrollable-overflow-region). Can be used to indicate
 there is content to scroll to in a given direction.
 
 Needs further exploration.
 
+### Scroll-direction
+
+Query based on direction of [scrolling](https://drafts.csswg.org/cssom-view/#scrolling). For instance:
+
+```css
+@container scroll-state(scroll-direction: top) {
+  .scrolling-up {
+    translate: 0 0;
+  }
+}
+```
+
+#### Workaround
+
+The current workaround solution to style elements based on scroll direction is by using `transition-delay` trick, see 
+[Solved by CSS Scroll-Driven Animations: hide a header when scrolling down, show it again when scrolling up](https://www.bram.us/2024/09/29/solved-by-css-scroll-driven-animations-hide-a-header-when-scrolling-up-show-it-again-when-scrolling-down/). However, in this approach, if a user was still pressing on the scrollbar after scrolling to the specific direction the `scroll-direction` would be considered as `none` since the scrollbar wasn't moving anymore. This behaviour is inconsistent with [the completed state of scrolling](https://drafts.csswg.org/cssom-view/#scroll-completed).
```

Comment:
In general, could you wrap the lines at roughly 80 chars like the rest of the doc?